BraveWallet/Extensions/WeiFormatter.swift

```diff
@@ -149,4 +149,14 @@ struct WeiFormatter {
     return (gwei * (BDouble(10) ** 9))
       .rounded().asString(radix: outputRadix.rawValue)
   }
+  
+  static func decimalToLamports(_ decimalString: String) -> UInt64? {
```

Comment:
Would be good to add unit test for it in `WeiFormatterTests`

Reply:
fixed in [85b7a06](https://github.com/brave/brave-ios/pull/5728/commits/85b7a069a9e3e7cc7b4d0571068e6fc5f898b95d)